### DISCUSSION TITLE
Broken Auth State - Can't log in/out

### DISCUSSION THREAD

**Discussion by eberling** [community/community#78815] [Copilot Conversations]
### Select Topic Area

Bug

### Body

Hi 🌻

Using VSCode, I had a subscription on one account, which expired. Now I want to use my company github account. But I cannot remove the old account, Output returns:

```
[INFO] [auth] [2023-12-12T12:44:21.405Z] Invalid copilot token: missing token: 403 
[ERROR] [default] [2023-12-12T12:44:21.407Z] Extension activation failed: "Thank you for using GitHub Copilot. Your subscription has ended. You are currently logged in as eberling."
```
What does not work:
- Clicking the tray icon and selecting "Sign In to Github" - nothing happens.
- Right-Clicking the profile icon in the left side bar, there is no sign out option. this is what I get: 
![image](https://github.com/community/community/assets/37538928/84407ef2-71fe-4dbc-878f-2af4c9413434)
- Uninstalling Github Copilot
- Deleting extension folder
- Using older versions

What I haven't tried yet: Buying a subscription in hopes that the auth server lets me get into a healthy auth state.

**Comment by aglaeaaaa** [comment]
Same here!
I'm assuming you're also using the trial version?

**Reply by c-eberling** [reply]
Yes, I had a trial. That expired a while ago though and left me in this limbo state...

**Comment by c-eberling** [comment]
heres a diagnostics log

## Copilot

- Version: 1.143.0
- Build: prod
- Editor: vscode/1.84.2

## Environment

- http_proxy: n/a
- https_proxy: n/a
- no_proxy: n/a
- SSL_CERT_FILE: n/a
- SSL_CERT_DIR: n/a
- OPENSSL_CONF: n/a

## Feature Flags

- Send Restricted Telemetry: unable to determine
- Custom Certificates: unable to determine

## Node setup

- Number of root certificates: 140
- Operating system: Darwin
- Operating system version: 22.6.0
- Operating system architecture: x64
- NODE_OPTIONS: n/a
- NODE_EXTRA_CA_CERTS: n/a
- NODE_TLS_REJECT_UNAUTHORIZED: n/a
- tls default min version: TLSv1.2
- tls default max version: TLSv1.3

## Network Configuration

- Proxy host: n/a
- Proxy port: n/a
- Proxy auth: n/a
- Kerberos SPN: n/a
- Reject unauthorized: disabled

## Reachability

- github.com: HTTP 200 - 
- copilot-proxy.githubusercontent.com: HTTP 200 - 
- api.githubcopilot.com: HTTP 200 - 
- default.exp-tas.com: HTTP 200 - 

## VS Code Configuration

- HTTP proxy: 
- HTTP proxy autentication: n/a
- Proxy Strict SSL: true
- Extension HTTP proxy support: override

## Extensions

- Is `win-ca` installed?: false
- Is `mac-ca` installed?: false
